InnerEye/ML/scalar_config.py

```diff
@@ -41,6 +42,7 @@ class EnsembleAggregationType(Enum):
 @unique
 class ScalarLoss(Enum):
     BinaryCrossEntropyWithLogits = "BinaryCrossEntropyWithLogits"
+    # WeightedCrossEntropyWithLogits does not work with 1 class_names, use 2 class_names and 2 outputs to allow weights
     WeightedCrossEntropyWithLogits = "WeightedCrossEntropyWithLogits"
     MeanSquaredError = "MeanSquaredError"
```

Comment:
For the case where labels are exclusive, we should have an option to use categorical cross entropy loss.